modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -66,7 +66,7 @@ if (!(process as any)?.browser) {
 
 const patchedRequestMethods = ['get', 'post', 'put', 'del', 'patch'] as const;
 
-export abstract class BitGoAPI implements BitGoBase {
+export class AbstractBitGoAPI implements BitGoBase {
```

Comment:
I think i may have been a bit unclear, I think we should just make this class `BitGoAPI` and take all the implementations you've added in `./bitgo.ts` and do them here. There shouldn't be a reason to have both an `AbstractBitGoAPI` and `BitGoAPI` class, is there?

Reply:
Makes sense, I agree with implementing it as one. I'll make the change right now.